ipamd/introspect.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ipamd
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	log "github.com/cihub/seelog"
+
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils"
+)
+
+const (
+	// IntrospectionPort is the port for ipamd introspection
+	IntrospectionPort = 51678
```

Comment:
Can you pick a different port than this?  51678 is used by the ECS agent and picking this port will prevent this plugin and ECS from ever being used together.

Reply:
@samuelkarp thanks and have opened #19 for this.